classes/options-general.php

```diff
@@ -264,17 +264,17 @@ function print_page() {
 
 									<div class="postbox">
 
-										<h3><?php _e( 'Location Defaults', 'simplemap' ); ?></h3>
+										<h3><?php _e( 'Location Defaults', 'SimpleMap' ); ?></h3>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->